Sources/TuistSupport/System/System.swift

```diff
@@ -198,6 +198,8 @@ public final class System: Systeming {
         "GEM_PATH", "RUBY_ENGINE", "GEM_ROOT", "GEM_HOME", "RUBY_ROOT", "RUBY_VERSION",
         // Xcode
         "DEVELOPER_DIR",
+        // Proxy
+        "HTTP_PROXY", "HTTPS_PROXY", "FTP_PROXY", "ALL_PROXY",
```

Comment:
I believe there's also a `NO_PROXY` option

Reply:
Good point! Will add it as well.